src/navigation/waypoint.py

```diff
@@ -82,6 +83,13 @@ def evaluate(self, ud) -> str:
                 else:
                     # We finished a waypoint associated with a fiducial id, but we have not seen it yet.
                     return WaypointStateTransitions.search_at_waypoint.name  # type: ignore
+            
+            if self.context.rover.stuck:
+                #Removed .name
+                rospy.logerr(f"TYPE: {type(WaypointStateTransitions.continue_waypoint_traverse.name)}\n")
```

Comment:
we shouldnt use logerr for things that aren't errors I know its hard to see otherwise but lets remember to remove this